LipidDyn/core.py

```diff
@@ -414,6 +414,8 @@ def calculate_density(self,
 
         Parameters
         ----------
+        selection : object
```

Comment:
this should be as the other docstrings: 

```
parameter_name : type
    description
```

so like 

```
selection : MDAnalysis.core.AtomGroup object
    selection of atoms on which density will be calculated
```
same for all the others